### DISCUSSION TITLE
Alternate name for API Variable

### DISCUSSION THREAD

**Discussion by james-rae** [fgpv-vpgf/fgpv-vpgf#4106] [Team Posts]
We had verbally discussed this earlier, but got lost in the grind.
The original API variable was `RV`, for RAMP Viewer.  When development on the new API started, `RZ` was picked as a placeholder name.  If we release `v3.0` as is, then `RZ` will be official.

Some suggestions for alternatives

- `RVI` - ramp viewer interface
- `RI` - ramp interface
- `RVA` - ramp viewer API
- `RA` - ramp API
- `RAE` - ramp API entryway, and just an overall great name
- `RAMP` - wordy but makes caller code more readable
- `RV2` - meh. would be weird if we ever made ramp 3

Other proposals welcome

**Comment by mweech** [comment]
Given what its for, I would suggest RAMP, but I am not attached to any particular option.  

Other options
-RYU - RAMP You Up
-RMA - RAMP Map API
-REFUND - RAMP Extensible Framework United N Decoupled
-REFACTOR - RAMP Extensible Framework And Cheers to OLD RAMP

**Comment by james-rae** [comment]
In the end we went with `RAMP`. 
Discussion is done, thanks.
